content/en/docs/concepts/services-networking/gateway.md

```diff
@@ -0,0 +1,250 @@
+is included here.
+
+<!-- body -->
+
+## Concepts
+
+The following concepts form the foundation of Gateway:
+
+* __Role-oriented:__ Gateway API resources are modeled after organizational roles that are
+  responsible for managing Kubernetes service networking:
+  * __Infrastructure Provider:__ Manages infrastructure that allows multiple isolated clusters
+    to serve multiple tenants, e.g. a cloud provider.
+  * __Cluster Operator:__ Manages clusters and is typically concerned with policies, network
+    access, application permissions, etc.
+  * __Application Developer:__ Manages an application running in a cluster and is typically
+    concerned with application-level configuration and [Service][service] composition.
+* __Portable:__ Gateway API specifications are defined as [custom resources][crds] and are supported
+  by many [implementations][implementations].
+* __Expressive:__ Gateway API resources support functionality for common traffic routing use cases
+  such as header-based matching, traffic weighting, and others that were only possible in [Ingress][ingress]
+  by using custom annotations.
+* __Extensible:__ Gateway allows for custom resources to be linked at various layers of the API.
+  This makes granular customization possible at the appropriate places within the API structure.
+
+Refer to the full Gateway API [documentation][gw-api] for additional details of these concepts.
+
+## Resource model
+
+Three primary resources exist in the Gateway API:
+
+* __GatewayClass:__ Defines a set of gateways with common configuration and managed by a controller
+  that implements the class.
+
+* __Gateway:__ Defines an instance of traffic handling infrastructure, e.g. cloud load balancer.
+
+* __Route:__  Route itself is not a resource but a general representation of different resources,
+  e.g. HTTPRoute, that define protocol-specific rules for mapping requests from a Gateway to resources
+  such as a Service.
+
+See the full Gateway API [documentation][gw-api] for additional details of all supported resources.
+
+### API kinds used for Gateway
+
+The Gateway API is organized into resources that have interdependent relationships to support the
+role-oriented nature of organizations. A Gateway resource is associated with a GatewayClass, which
+describes the gateway controller responsible for managing Gateways of this class. Route resources,
+e.g. HTTPRoute, are then associated to Gateway resources, which define protocol-specific rules for
+mapping requests from a Gateway to backend resources. A Gateway can filter the routes that may be
+attached to its `listeners`, forming a bidirection trust model with Routes.
+
+The following figure illustrates the relationships of the three primary Gateway API resources:
+
+{{< figure src="/docs/images/gateway-api-schema.svg" alt="gateway-api-schema" class="diagram-medium" link="https://raw.githubusercontent.com/kubernetes-sigs/gateway-api/main/site-src/images/schema-uml.svg" >}}
```

Comment:
- You must commit the images here and serve them via our CDN. Don't rely on the `main` branch of the Gateway API having the right content.
- Please set a useful `alt` attribute.